query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/config/LdapConfigurationField.java

```diff
@@ -132,11 +155,10 @@ public LdapConfigurationField mapAllClaims(Map<String, String> mapping) {
 
   @Override
   public List<Field> getFields() {
-    return ImmutableList.of(pid, connection, bindUserInfo, settings, claimMappings);
+    return ImmutableList.of(pid, connection, loadBalancing, bindUserInfo, settings, claimMappings);
   }
 
   public LdapConfigurationField useDefaultRequired() {
-    connection.useDefaultRequired();
```

Comment:
❗️ Any reason this was removed? You can still call `useDefaultRequired` on a list.

Reply:
I updated the field type and added this call back in